lib/config/target.js

```diff
@@ -60,8 +62,21 @@ const versionDependent = (major, minor) => {
 	};
 };
 
+// cspell:word browserslist
+
 /** @type {[string, string, RegExp, (...args: string[]) => TargetProperties][]} */
 const TARGETS = [
+	// todo find out how we can do this thru a plugin
+	[
+		"browserslist[:[///path-to-config][>/path-to-browserslist][?query][:browserslist-env]]",
```

Comment:
This could be simplified: `browserslist / browserslist:env / browserslist:query / browserslist:path-to-config / browserslist:path-to-config:env`

path-to-config can be detected because it's an absolute path

Between `env` and `query` can be decided via `Object.keys(browserslist.findConfig())` (maybe with some extra checks)

So only `:` is used and nobody need to remember which sign to use...

Reply:
done